cli/standalone.rs

```diff
@@ -0,0 +1,114 @@
+use crate::colors;
+use crate::flags::Flags;
+use crate::permissions::Permissions;
+use crate::program_state::ProgramState;
+use crate::tokio_util;
+use crate::worker::MainWorker;
+use deno_core::error::AnyError;
+use deno_core::futures::FutureExt;
+use deno_core::ModuleLoader;
+use deno_core::ModuleSpecifier;
+use deno_core::OpState;
+use std::cell::RefCell;
+use std::convert::TryInto;
+use std::env::current_exe;
+use std::fs::File;
+use std::io::Read;
+use std::io::Seek;
+use std::io::SeekFrom;
+use std::pin::Pin;
+use std::rc::Rc;
+
+pub fn standalone() {
+  let current_exe_path =
+    current_exe().expect("expect current exe path to be known");
+
+  let mut current_exe = File::open(current_exe_path)
+    .expect("expected to be able to open current exe");
+  let magic_trailer_pos = current_exe
+    .seek(SeekFrom::End(-12))
+    .expect("expected to be able to seek to magic trailer in current exe");
+  let mut magic_trailer = [0; 12];
+  current_exe
+    .read_exact(&mut magic_trailer)
+    .expect("expected to be able to read magic trailer from current exe");
+  let (magic_trailer, bundle_pos) = magic_trailer.split_at(4);
+  if magic_trailer == b"DENO" {
```

Comment:
Maybe make a bit more magic? Also suggest to make it a `const` at the top of the file